random/rand_source_chacha.mbt

```diff
@@ -0,0 +1,469 @@
+// Copyright 2024 International Digital Economy Academy
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
is it easy to move it into random/internal/source_chacha?

Reply:
Done